src/common/Throttle.cc

```diff
@@ -222,31 +226,34 @@ int64_t Throttle::put(int64_t c)
     return 0;
   }
 
-  ceph_assert(c >= 0);
+  ceph_assert(c != 0);
```

Comment:
why?

Reply:
@tchaikov .  it's my fault, I meated to avoid checking "c==0".